deps/rabbitmq_amqp1_0/src/rabbit_amqp1_0_reader.erl

```diff
@@ -435,13 +439,30 @@ handle_1_0_connection_frame(_Frame, State) ->
     maybe_close(State#v1{connection_state = closing}).
 
 handle_1_0_session_frame(Channel, Frame, State) ->
-    case get({channel, Channel}) of
-        {ch_fr_pid, SessionPid} ->
+    case find_session_pid_for_channel(Channel, State) of
+        closing ->
```

Comment:
I believe never. That clause was there before this PR. I searched for possible situations when this record may have the closing value but I did not find any. I did not want to delete it just in case.